docs/content/_navigation.json

```diff
@@ -741,10 +741,14 @@
             "title": "Structuring your Dagster project",
             "path": "/guides/dagster/recommended-project-structure"
           },
+          {
+          "title": "Building machine learning pipelines",
+          "path": "/guides/dagster/ml-pipeline"
+          },
           {
             "title": "Using environment variables and secrets",
             "path": "/guides/dagster/using-environment-variables-and-secrets"
-          },
+          },          
```

Comment:
I think we don't need to add this whitespace

Reply:
removed